launcher/CMakeLists.txt

```diff
@@ -568,6 +568,23 @@ set(LOGIC_SOURCES
     ${ATLAUNCHER_SOURCES}
 )
 
+ecm_qt_declare_logging_category(LOGIC_SOURCES
+    HEADER prismlauncherdebug.h
```

Comment:
I would prefer if this was just called `launcherlog.h`. We have some downstream forks with different branding and don't want to make their lives a living hell